app/src/main/java/com/beemdevelopment/aegis/ui/views/EntryAdapter.java

```diff
@@ -198,6 +204,14 @@ private void updateShownEntries() {
             }
         }
 
+        for (DatabaseEntry entry: _entries) {
+            if (_searchFilter != null) {
+                if (!entry.getIssuer().toLowerCase().contains(_searchFilter.toLowerCase())) {
```

Comment:
This should be moved to ``isEntryFiltered``, so that newly added entries that do not match the filter are also not shown.